docker-storage-setup.sh

```diff
@@ -1199,6 +1197,17 @@ fi
 if [ -e /etc/sysconfig/docker-storage-setup ]; then
   source /etc/sysconfig/docker-storage-setup
 fi
+if [ ! -z "$DOCKER_ROOT_VOLUME" ]; then
+    Info "DOCKER_ROOT_VOLUME is deprecated, and will be removed soon. Use CONTAINER_ROOT_LV_MOUNT_PATH instead."
+fi
```

Comment:
This check `if [ ! -z "$DOCKER_ROOT_VOLUME" ]` is already handled in `check_storage_options()`.
https://github.com/projectatomic/docker-storage-setup/blob/master/docker-storage-setup.sh#L1032